server/views/pages/view/cases.njk

```diff
@@ -132,11 +135,11 @@
             ],
             rows: licences
         }) }}
-    {% else %}
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-three-quarters">
-                <h3 class="govuk-heading-s">There are no licences to print which match the search criteria.</h3>
+        {% else %}
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-three-quarters">
+                    <h3 class="govuk-heading-s">There are no licences to print which match the search criteria.</h3>
+                </div>
             </div>
-        </div>
-    {% endif %}
-{% endblock %}
+        {% endif %}
+    {% endblock %}
```

Comment:
meh.. missing newline at end of file

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline